### PR TITLE
Merge back Az.storage 1.13.1-preview

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by blueww** [comment]
@dingmeng-xue 
The 2 *.json file should NOT be merged to preview branch.
1. They are for breaking change tracking, and we don't need to track breaking change in preview release.
2. it will take a lot of trouble when merge back change in master branch to storage preview branch.

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 1.13.2
```
Comment (blueww) [inline src/Storage/Storage.Management/ChangeLog.md]: should be 1.13.3

File: src/Storage/Storage.Management/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -46,5 +46,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("1.13.1")]
-[assembly: AssemblyFileVersion("1.13.1")]
+[assembly: AssemblyVersion("1.13.2")]
```
Comment (blueww) [inline src/Storage/Storage.Management/Properties/AssemblyInfo.cs]: should be 1.13.3

**Pull request merged**